workers/varcode_annotator.py

```diff
@@ -132,7 +132,10 @@ def get_varcode_annotations(genotypes, vcf_id, ensembl_release_num):
         best_effect = variant.effects().top_priority_effect()
         gene_name = best_effect.gene_name
         transcript = best_effect.transcript_id
-        notation = best_effect.short_description
+        if best_effect.__class__.__name__ == "Intragenic":
```

Comment:
I'd use `isinstance(best_effect, Intragenic)` (if that works), instead. Does Intragenic have no short_description? Is that something that should be fixed in varcode? (If so, file an issue?)